sympy/stats/stochastic_process_types.py

```diff
@@ -148,8 +151,8 @@ def joint_distribution(self, *args):
             return JointDistribution(*args)
         # TODO: Add tests for the below part of the method, when implementation of Bernoulli Process
         # is completed
-        pdf = Lambda(*[arg.name for arg in args],
-                expr=Mul.fromiter(arg.pspace.distribution.pdf(arg) for arg in args))
+        pdf = Lambda(tuple(args),
+                    expr=Mul.fromiter(arg.pspace.process.pmf(arg) for arg in args))
```

Comment:
I think using pdf or pmf depends on the type(discrete or continuous). You can use `__getattribute__` to fetch the correct(pdf or pmf). A minor change is needed here.

Reply:
Can you please explain the change?

Reply:
```
pdf = Lambda(tuple(args),
                    expr=Mul.fromiter(arg.pspace.process.density(arg) for arg in args))
```
And In the BernoulliProcess class:
```
def __getattribute__(self, name):
        if name == 'density':
           return self.pmf
        else:
            return object.__getattribute__(self, name)
```
Is this change required?

Reply:
Do,
```python
density = Lambda(tuple(args),
                    expr=Mul.fromiter(arg.pspace.process.density(arg) for arg in args))
```